### PR TITLE
docs(refs T30969): add checkbox documentation

### PR DESCRIPTION
**Pull request by hwiem**
Moves `DpCheckbox` to the `components` root and adds some basic documentation.

(Also, fixes a duplicated import statement)

Related tickets: https://yaits.demos-deutschland.de/T30969, https://yaits.demos-deutschland.de/T30976

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,9 @@
 
 ## UNRELEASED
 
+- docs: add documentation for DpCheckbox
+- refactor: Move DpCheckbox to components root
```
Comment (spiess-demos) [inline CHANGELOG.md:6]: I think the changelog may not contain implementation changes. In this case, it does not matter if we move components around as long as they are exported from the top level `index.js`.

File: src/components/DpCheckbox/DpCheckbox.stories.mdx [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  title="Components/Checkbox"
+  component={DpCheckbox} />
+
+# Checkbox
+
+The `DpCheckbox` component renders an input of type `checkbox`. Use it for a single option that the user can turn on or
+off.
+
+In order to display several checkboxes grouped together, use `DpCheckboxGroup`. It allows you to define a group
+label and have checkbox labels with normal `font-weight`.
```
Comment (spiess-demos) [inline src/components/DpCheckbox/DpCheckbox.stories.mdx:14]:
```suggestion
In order to display several checkboxes grouped together, use `DpCheckboxGroup`. 
It allows for defining an additional group label and have checkbox labels with normal `font-weight`.
```

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (hwiem) [reply CHANGELOG.md]: removed in c5025cdc4b90074f4998b25225a33e8984550a3b

**Review by muellerdemos — APPROVED** [review]
even though I have not worked with storybook before, this lgtm and also helps me how to work with storybooks

**Review by spiess-demos — APPROVED** [review]
lgtm!

**Pull request merged**